docs/migration.md

```diff
@@ -418,11 +404,86 @@ exporters:
 
 ##### Collection should begin
 
-The OpenTelemetry Collector doesn't have a substitute for this Installed Collector option.
-It supports two options, starting at the beginning or end of a file.
-Starting at the `beginning` will read the entire file every time it's started.
-Starting at the `end` will read only logs appended to file after it's started.
-This is configurable with the `start_at` option.
+The OpenTelemetry Collector substitution for this Installed Collector option is complex,
+as the following steps needs to be performed:
+
+- reading logs using [Filelog Receiver][filelogreceiver]
+- parsing timestamp from the logs using [Logs Transform Processor][logstransformprocessor]
```

Comment:
Isn't it possible to do all this in one step using the Filter processor with [OTTL](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/filterprocessor#ottl)? The specific example of only logs from 2023 would be something like:

```yaml
processors:
  filter:
    logs:
      log_record:
      - 'Int(Substring(body, 0, 4)) >= 2023'
```

Even if we want the condition to be more complex, I believe we should be able to do away without adding additional attributes and removing in a separate step?

Reply:
damn, I was scrooling README and stopped on `expr` :facepalm:. We definitely should use OTTL in filterprocessor if it support its. I used simple example, but can easily imagine equivalent:
`let's take logs from 2022-12-04 13:13:15`

Reply:
refactored